diff --git a/.SRCINFO b/.SRCINFO
index 3240644..f8a9854 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -1,6 +1,6 @@
 pkgbase = python-pefile
 	pkgdesc = Python PE parsing module
-	pkgver = 2017.11.5
+	pkgver = 2018.8.8
 	pkgrel = 1
 	url = https://github.com/erocarrera/pefile
 	arch = any
@@ -9,8 +9,8 @@ pkgbase = python-pefile
 	makedepends = python2
 	makedepends = python-future
 	makedepends = python2-future
-	source = python-pefile-2017.11.5.tar.gz::https://github.com/erocarrera/pefile/archive/v2017.11.5.tar.gz
-	sha512sums = 820f7770d18256857436698a9d72276dc8089a2a85c20f1c2e284c541680879016f7acf7b3029e7dd1f32ef093b52fe1c8e8f7962e5b9a7dbbc0f94caf6d0d66
+	source = https://github.com/erocarrera/pefile/releases/download/v2018.8.8/pefile-2018.8.8.tar.gz
+	sha512sums = 44e8ed40f03e795badcc17cc6567165b3430aa80034f1b02b35fc15c94ce6b544ac195dedcea5858664c169fc4241e999df09a36f1a8a6635ce536af40284d29
 
 pkgname = python-pefile
 	depends = python
diff --git a/PKGBUILD b/PKGBUILD
index 4ced10d..92b3dd9 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,15 +3,15 @@
 
 pkgbase=python-pefile
 pkgname=('python-pefile' 'python2-pefile')
-pkgver=2017.11.5
+pkgver=2018.8.8
 pkgrel=1
 pkgdesc="Python PE parsing module"
 arch=('any')
 url="https://github.com/erocarrera/pefile"
 license=('MIT')
 makedepends=('python' 'python2' 'python-future' 'python2-future')
-source=("${pkgbase}-${pkgver}.tar.gz::https://github.com/erocarrera/pefile/archive/v${pkgver}.tar.gz")
-sha512sums=('820f7770d18256857436698a9d72276dc8089a2a85c20f1c2e284c541680879016f7acf7b3029e7dd1f32ef093b52fe1c8e8f7962e5b9a7dbbc0f94caf6d0d66')
+source=("https://github.com/erocarrera/pefile/releases/download/v${pkgver}/pefile-${pkgver}.tar.gz")
+sha512sums=('44e8ed40f03e795badcc17cc6567165b3430aa80034f1b02b35fc15c94ce6b544ac195dedcea5858664c169fc4241e999df09a36f1a8a6635ce536af40284d29')
 
 prepare() {
   cd "${srcdir}"
